_projects/2023/radis/optimizing_radis_app.md

```diff
@@ -0,0 +1,93 @@
+---
+ name: Optimizing Radis app
+ desc: Our project is all about enhancing user experience to the next level! We're committed to bringing you cutting-edge features and  fine-tuning these features for maximum performance and efficiency, and rigorous testing to ensure they meet the needs of our highly valued end users.
+
+# add a short one line description of your project
+
+requirements:
+
+# Student requirements
+
+- TypeScript
+- React
+- FastApi
+- Testing (pytest && vitest)
+
+difficulty: medium
+issues:
+
+- <https://github.com/suzil/radis-app/issues/669>
+- <https://github.com/suzil/radis-app/issues/85>
+- <https://github.com/suzil/radis-app/issues/674>
+ mentors:
```

Comment:
I think here and below you either need the empty line, and/or remove the leading space.